nodejs/week2/src/backend/routes/meals.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+// const app = express();
+const router = express.Router()
+
+const mealsArray = require('../data/meals.json');
+
+     
+// ++++++++++++++++++++++++ Testing for combining the whole meal search commands into one ++++++++++++++++++++++++  
+
+router.get('/meals', (req, res)=>{

```

Comment:
Usually, it's a nice practice to follow naming conventions 😉
You have violations there - naming from a capital letter and not using camel. Also, code alignment is broken in several places. Consider improving it next time 🙂